hr_employee_operating_unit/models/hr_employee.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2017 Genweb2 Limited - Matiar Rahman
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models
+
+
+class HREmployee(models.Model):
+
+    _inherit = 'hr.employee'
+
+    @api.model
+    def _get_operating_unit(self):
+        user = self.env['res.users'].browse(self._uid)
+        return user.default_operating_unit_id
+
+    @api.model
+    def _get_operating_units(self):
```

Comment:
I suppose you can get rid of this method and call directly `_get_operating_unit` ?